<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocDialog</class>
 <widget class="QDialog" name="LocDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Location</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="27" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="C" country="AnyCountry"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>590</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="btnIcon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="locale">
          <locale language="C" country="AnyCountry"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="checkName">
         <property name="title">
          <string>Name</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinNameX">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>-9999</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinNameY">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>-9999</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelNameX">
            <property name="text">
             <string>X</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>spinNameX</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelNameY">
            <property name="text">
             <string>Y</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>spinNameY</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QToolButton" name="btnColorName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="locale">
             <locale language="C" country="AnyCountry"/>
            </property>
            <property name="text">
             <string>Color</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QToolButton" name="btnFont">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="locale">
             <locale language="C" country="AnyCountry"/>
            </property>
            <property name="text">
             <string>Font</string>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEditName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="locale">
             <locale language="C" country="AnyCountry"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="checkZrv">
         <property name="title">
          <string>Zone</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditWidth"/>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QToolButton" name="btnColorZrv">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="locale">
             <locale language="C" country="AnyCountry"/>
            </property>
            <property name="text">
             <string>Color</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Width</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditWidth</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkZrvEnable">
            <property name="text">
             <string>Enabled</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditAzimuth"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Azimuth</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditAzimuth</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Range</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditRange</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditRange"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QGroupBox" name="checkZrvLines">
         <property name="title">
          <string>Zone contour</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinLines">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelWidth">
            <property name="text">
             <string>Width</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>spinLines</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QToolButton" name="btnColorLines">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="locale">
             <locale language="C" country="AnyCountry"/>
            </property>
            <property name="text">
             <string>Color</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupCoords">
         <property name="title">
          <string>Coords</string>
         </property>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Latitude</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditLat</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditLat"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Longitude</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditLon</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditLon"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Altitude</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>lineEditHeight</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditHeight"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="27" column="0">
    <widget class="QPushButton" name="btnDefault">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnDefault</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LocDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LocDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
